applications/fluid_structure_interaction/cylinder_with_flag/application.h

```diff
@@ -178,19 +200,20 @@ class Application : public FluidFSI::ApplicationBase<dim, Number>
 
     // PHYSICAL QUANTITIES
     param.start_time = 0.0;
-    param.end_time   = END_TIME;
+    param.end_time   = END_TIME * end_time_factor;
```

Comment:
Should we really introduce a factor for the time via the input file, but keep the `END_TIME` as a separate variable? I would prefer to specify the actual final time, because setting information in two places is usually confusing.